core/src/main/java/com/scalar/db/storage/jdbc/query/SimpleSelectQuery.java

```diff
@@ -127,6 +186,17 @@ private String orderBySqlString() {
             .collect(Collectors.joining(","));
   }
 
+  private String relationalOrderBySqlString() {
+    if (orderings.size() == 0) {
```

Comment:
[minor] I think `if (orderings.isEmpty())` would be slightly clearer.

Reply:
Fixed in 1490ed6.